src/cogent3/app/data_store_new.py

```diff
@@ -173,6 +173,14 @@ def unique_id(self):
     def source(self):
         return self.unique_id
 
+    @property
+    def md5(self):
+        return self.data_store.md5(self.unique_id)
+
+    def write_to_logger(self, logger):
```

Comment:
Doesn't belong on the DataMember. This is wrapping a logging method, revert this and resurrect the original code inside the loop.